cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/type/impl/CassandraCalendarDateType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.cassandra.type.impl;
+
+import org.hibernate.MappingException;
+import org.hibernate.engine.spi.Mapping;
+import org.hibernate.ogm.type.impl.AbstractGenericBasicType;
+import org.hibernate.type.descriptor.java.CalendarDateTypeDescriptor;
+
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * Cassandra doesn't do Calendar, but it does do Date, so convert them.
```

Comment:
> Internal storage in cassandra 2.1 is actually long from epoch, just as inside java's Date [...] We're sending it longs instead, so handling the TZ is our problem

Yes, that's what I mean, couldn't we pass String (and store) instead in order to convey the TZ information? I find silently loosing the TZ information from calendars a bit troubling. At least a warning should be raised, that `Calendar` properties can be stored but will be read back without the TZ information.